src/main.rs

```diff
@@ -21,14 +22,15 @@ fn main() -> Result<()> {
                     break repo;
                 }
             }
-            None => bail!("Test"),
+            None => bail!("Not in a repo"),
         }
     };
 
     let config_url = repo.get_remote_url();
 
     if let Some(url) = &config_url {
         if config.ignores(url) {
+            println!("Abort: Repo on ignore list");
```

Comment:
Could you remove this, so this just exits silently if one is actively ignoring the repository?